ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/logging.cfg

```diff
@@ -1,7 +1,10 @@
 !
+no logging repeat-messages
 logging buffered 1000000 warnings
 no logging trap
 logging console errors
+logging event congestion-drops interval 10
+logging event link-status global
```

Comment:
this is not correct order of config compare to actual EOS cli. For `logging event congestion-drops interval 10` we need to create new j2 module and include it in eos-intended-config.j2

Reply:
I noticed there are other parts of the logging schema that are not rendered into the correct spot in the EOS configuration. As an example the existing `logging event storm-control` looks like it should be rendered after `eos/aliases.j2` and before `eos/daemon-terminattr.j2` to match the ordering of the EOS configuration. While fixing my own additions should I look to bring the rest of the logging data model into compliance here? If so under this PR or a new one?

Reply:
The reordering of existing stuff is covered in this PR - https://github.com/aristanetworks/avd/pull/4449/

Reply:
Yes so just fix the order of your changes, others will be handle in above PR.

Reply:
Done